commands

```diff
@@ -2,33 +2,52 @@
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 source "$(dirname $0)/../common/functions"
 
+is_webhook() {
+  echo "$1" | grep '^http'
```

Comment:
Would this not also match if an app name begins with `http`, for example `httpserver`?